packages/default/scss/pdf-viewer/_layout.scss

```diff
@@ -33,61 +33,201 @@
             background: none;
             overflow: visible;
         }
+    }
 
 
-        // Canvas
-        .k-canvas {
-            display: flex;
-            flex-direction: column;
-            flex: 1 1 auto;
-            outline: none;
+    // Canvas
+    .k-pdf-viewer-canvas {
```

Comment:
Won't this change affect teams other than the pilot?

Reply:
I checked this. The rendering in all suites includes both classes at the same element `.k-canvas.k-pdf-viewer-canvas`. In the fluent theme we have used the `k-pdf-viewer-canvas` class for the CSS rules but in the default theme we have used the `k-canvas`. I made this change for consistency.

Here is the reference: https://github.com/telerik/kendo-themes/pull/5173/files/20a40e24f6584be88d494f81200ac2e9bbf63d35#diff-ecb5c4c15b8f8127171653f1208736404699fdaa31ce73207386069056b8558eL40

This said, there would be no affected team from this change :)